src/sources.js

```diff
@@ -242,7 +242,7 @@ const sources = {
         description: 'pubmed comprises more than 29 million citations for biomedical literature from medline, life science journals, and online books. citations may include links to full-text content from pubmed central and publisher web sites',
```

Comment:
Since the number of citations keep increasing, I suggest we also change the description.
From the about page: "PubMed is a free resource supporting the search and retrieval of biomedical and life sciences literature with the aim of improving health–both globally and personally."